### DISCUSSION TITLE
Welcome to PicMo Discussions!

### DISCUSSION THREAD

**Discussion by joeattardi** [joeattardi/picmo#185] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guleswine** [comment]
I would like to see more practical code examples. For example, I wanted to use poupup in a Laravel application, on a page with many text fields, but so far without success (

**Reply by joeattardi** [reply]
I'm not familiar with Laravel or how it works, sorry. If you do end up getting it working you can always submit a PR that adds a Laravel example :)

I can try to help if you give some indication of what issues you're having?

**Reply by joeattardi** [reply]
Also, please create a new top level discussion or open an issue with details for situations like this.
